tensorflow/core/common_runtime/eager/mkl_eager_op_rewrite.cc

```diff
@@ -88,6 +88,7 @@ REGISTER_REWRITE(EagerOpRewriteRegistry::PRE_EXECUTION, MklEagerOpRewrite);
 // Constructor
 MklEagerOpRewrite::MklEagerOpRewrite(string name, string file, string line)
     : EagerOpRewrite(name, file, line) {
+  registered_kernels_map_ = std::unordered_map<std::string, bool>();
```

Comment:
Then isn't it better to initialize it in the : line? Line, MklEagerOpRewrite::MklEagerOpRewrite(string name, string file, string line)
    : EagerOpRewrite(name, file, line), registered_kernel_map_() { ...

?

Reply:
That is right. I will do that.